Help/FAQ.rst

```diff
@@ -20,6 +20,22 @@ In short coala does two things:
   everything but the actual analyzer routine. Write just the parameters
   your custom analysis needs and the analysis part, we'll take care of the rest
 
+What is a bear ?
+----------------
+
+A bear is meant to do static analysis on source code. It can check your code
+for potential problems, calculate metrics and even provide corrections for
+your code. Bears analyze the source code provided by coala.
+
+Each bear has its own function and supported languages. So for different
+purposes you can use different bears.
+
+Find out more about bears
+`here <http://api.coala.io/en/latest/Developers/Writing_Native_Bears.html>`__.
```

Comment:
I think this should be "Find out more about writing bears"

And there should be a "Run your first bear!" or something like that pointing to https://docs.coala.io/en/latest/Users/Tutorial.html